pfcp/handler/handler.go

```diff
@@ -131,6 +131,16 @@ func HandlePfcpAssociationSetupResponse(msg *pfcpUdp.Message) {
 		}
 		logger.PfcpLog.Infof("Handle PFCP Association Setup Response with NodeID[%s]", nodeID.ResolveNodeIdToIp().String())
 
+		//Get NodeId from Seq:NodeId Map
+		seq := msg.PfcpMessage.Header.SequenceNumber
+		nodeID = pfcp_message.FetchPfcpTxn(seq)
```

Comment:
Should the variable be trans or something? nodeId does not seem correct. Does this function returns nodeId of UPF ?

Reply:
It returns NodeId